### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by MalloZup**
## What does this PR?

We need a changelog to keep track of what we add.

This pr aims to add all features of our initial `0.0.3` version.

I have added a shared branch, so feel free to commit there.

Take inspiration from this https://keepachangelog.com/en/1.0.0/

and https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/CHANGELOG.md

### CODE REVIEW

**Comment by MalloZup** [comment]
update all features here: https://github.com/kubic-project/kubic-init/releases/tag/untagged-0d2dc1d702f00c4955be so we can use that as wiki

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+## 0.0.5 Initial kubic-init
```
Comment (MalloZup) [inline CHANGELOG.md]: i did a typo here, it should be `0.0.3`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- Upgrade to k8s 1.13.0
+- Use the v1beta1 API
+- Use leap15 as docker image
+- Use the etcd image from the registry.suse.de
+- Use admin kubeconfig to deploy CRD's
+- Use a higher level kubeadm API and get rid of our custom stages. Get some config from environment variables.      
+- Honor the DynamicKubeletConfig feature gate
+
+
+### CI
```
Comment (MalloZup) [inline CHANGELOG.md]: I think CI stuff we can omit this in the changelog. Since what we care is more about the `features/improvements` etc

Comment (nirmoy) [reply CHANGELOG.md]: @MalloZup please recheck, I might've/surely  missed some. Also we should squash before merging this PR

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- When `autoApproval=false` in the config file, remove the RBAC rules used for approvinfg nodes automatically.
+- Use leap15 as docker image
+- Use the etcd image from the registry.suse.de
+
+### K8s & Kubeadm
+- Add a command for getting kubeadm's version and log it.
+- Upgrade to k8s 1.13.0
+- Use the v1beta1 API
+- Use admin kubeconfig to deploy CRD's
+- Use a higher level kubeadm API and get rid of our custom stages. Get some config from environment variables.
```
Comment (inercia) [inline CHANGELOG.md]:
```suggestion
- Use a higher level kubeadm API. Get some config from environment variables.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
+## 0.0.3 Initial kubic-init
+### Kubic-init
+- Check if the kubic-init must reset or bootstrap before going forward
+- cni: make default cni config placement configurable
+- cni: add cilium cni plugin
+- Fix Flannel initialization and make the CNI configuration and binaries dirs configurable
```
Comment (inercia) [inline CHANGELOG.md]:
```suggestion
- cni: flannel initialization and make the CNI configuration and binaries dirs configurable
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- config: parse bind interface
+- Use Flannel as default CNI
```
Comment (inercia) [inline CHANGELOG.md]:
```suggestion
- cni: use flannel as the default CNI driver
```

File: CHANGELOG.md (hunk 2 above)
Comment (MalloZup) [reply CHANGELOG.md]: yop

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+
+## 0.0.3 Initial kubic-init - January 12 2019
+
+### Kubic-init
+
+- Add check if the kubic-init must reset or bootstrap before going forward (https://github.com/kubic-project/kubic-init/pull/149)
```
Comment (MalloZup) [inline CHANGELOG.md]: @inercia @nirmoy i think we should add in future also the PR link like here (httpsblabl/132).

Now for this `first` release, i think we can be more flexible. However feel free to addo here and there some links of prs of the changelog-entry, so we have more data.

I am also ok if we don't do it for this first release.

I have merged all comments. I think we are ok. What is your pov?

Comment (inercia) [reply CHANGELOG.md]: I'm fine with the current status of this PR. I think it is more than enough for a first version of the Changelog. Thanks foro dealing with this chore, @MalloZup .

Comment (MalloZup) [reply CHANGELOG.md]: thx to @nirmoy and you :sunflower:

**Pull request merged**